src/librustc_target/spec/linux_musl_base.rs

```diff
@@ -46,7 +46,7 @@ pub fn opts() -> TargetOptions {
     base.has_elf_tls = true;
     base.has_rpath = true;
     base.position_independent_executables = true;
-    base.relro_level = true;
+    base.relro_level = RelroLevel::Full;
```

Comment:
Also change import to include `spec::{LinkerFlavor, TargetOptions, RelroLevel}`;